backend/app/services/report_service.py

```
@@ -1,132 +1,189 @@
+    
+async def add_review(db: AsyncSession, review: ReportReviewUpdate, id_report: str):
+    result = await db.execute(select(Report).where(Report.id_report == id_report))
+    report = result.scalar_one_or_none()
+    if not report:
+        raise ValueError("Report not found")
+    report.review_score = review.review_score
+    report.review_text = review.review_text
+    await db.commit()
+
+    # Fire-and-forget: send feedback to LangSmith without blocking or failing the endpoint
+    asyncio.create_task(_send_feedback_to_langsmith_wrapper(id_report, review.review_score, review.review_text))
```

Ruff (0.15.14)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)